auth/src/main/java/software/amazon/awssdk/auth/AwsExecutionAttributes.java

```diff
@@ -17,10 +17,11 @@
 
 import software.amazon.awssdk.annotations.ReviewBeforeRelease;
 import software.amazon.awssdk.auth.credentials.AwsCredentials;
+import software.amazon.awssdk.auth.signer.internal.AwsSignerParams;
 import software.amazon.awssdk.core.interceptor.ExecutionAttribute;
 import software.amazon.awssdk.core.interceptor.ExecutionInterceptor;
 import software.amazon.awssdk.core.interceptor.SdkExecutionAttributes;
-import software.amazon.awssdk.core.runtime.auth.Signer;
+import software.amazon.awssdk.core.signerspi.Signer;
```

Comment:
Any particular reason to include spi in the package name?

Reply:
Initially I tried to keep signer interfaces in a separate module and named it spi suffix. Later I moved it to core as I needed to use class in core. Didn't change the name during the transition. Fixed now.